packages/vue-virtual/src/index.ts

```diff
@@ -103,7 +103,8 @@ export function useWindowVirtualizer<TItemElement extends Element>(
   return useVirtualizerBase<Window, TItemElement>(
     computed(() => ({
       // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      getScrollElement: () => (typeof Window !== 'undefined' ? window : null!),
```

Comment:
this would have always evaluated to `null` since it should have been lowercase `window` instead of `Window`